### PR TITLE
Document the additional_directives parameter

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -3,91 +3,96 @@
...
+#                            	type:Boolean
 #
-# $additional_options::    Additional options
-#                          type:Hash[String, String]
+# $additional_options::      	Additional options
+#                            	type:Hash[String, String]
+#
+# $additional_directives::  	Additional directives. These are free form
+# 				strings that allow for full customization. Use
+# 				with caution.
```
Comment (stbenjam) [inline manifests/init.pp]: Alignment here needs fixing

**Comment by mmoll** [comment]
also, please rebase for the latest modulesync :)

**Comment by ekohl** [comment]
Done, forgot to place my .vimrc on my new laptop and didn't have tab expansion :(

**Comment by mmoll** [comment]
merged, bedankt @ekohl!

**Pull request merged**